### PR TITLE
Added support for multiple authentication types per operation.

### PR DESCRIPTION
**Pull request by malteburkert**
This change solves issue https://github.com/yonaskolb/SwagGen/issues/214

### CODE REVIEW

File: Templates/Swift/Sources/APIService.swift [hunk 1]
```diff
@@ -8,16 +8,16 @@ public struct APIService<ResponseType: APIResponseValue> {
     public let path: String
     public let hasBody: Bool
     public let isUpload: Bool
-    public let securityRequirement: SecurityRequirement?
+    public let securityRequirements: [SecurityRequirement]?
 
-    public init(id: String, tag: String = "", method:String, path:String, hasBody: Bool, isUpload: Bool = false, securityRequirement: SecurityRequirement? = nil) {
+    public init(id: String, tag: String = "", method:String, path:String, hasBody: Bool, isUpload: Bool = false, securityRequirements: [SecurityRequirement]? = nil) {
```
Comment (yonaskolb) [inline Templates/Swift/Sources/APIService.swift]: Maybe we could make this a non optional that defaults to `[]`?

**Comment by yonaskolb** [comment]
Thanks @malteburkert. Could you also please add a changelog entry

File: Templates/Swift/Sources/APIService.swift (hunk 1 above)
Comment (malteburkert) [reply Templates/Swift/Sources/APIService.swift]: Sure, I am going to supply this change immediately.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 ### Fixed
 - Fixed `'??' has non-optional type` warning #207
 - Fixed incorrect replacements in server variables #209
+- Fixed missing support for multiple authentication types per operation #214 #222 @malteburkert
+- Fixed nullable references not being generated as optionals #216 @alephao
```
Comment (yonaskolb) [inline CHANGELOG.md]: You’ll need to rebase or merge master in. PR #216 is in 4.3.1 that has since been released

**Comment by benrudhart** [comment]
Hi there,

thank you @malteburkert for creating this MR. Since I need this feature as well I'd like to ask if I can help to get this PR merged?

**Comment by malteburkert** [comment]
@yonaskolb could you please merge this PR? thanks in advance

**Pull request merged**